challenges/day6-issues/hard-task-5-image-gallery/index.html

```
@@ -0,0 +1,45 @@
+                <option value="food">Food</option>
+            </select>
+        </div>
+
+        <div class="gallery" id="gallery"></div>
+    </div>
+
+    <!-- Lightbox Modal -->
+    <div id="lightbox" class="lightbox">
+        <span class="close">&times;</span>
+        <div class="lightbox-content">
+            <img id="lightboxImg" src="" alt="">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)